### PR TITLE
Initial upload of velocity gauge part and be familiar with PR upload steps.

### PR DESCRIPTION
**Pull request by HeFuxiang94**

### CODE REVIEW

**Comment by Satinelamp** [comment]
Does this PR only include a part of the solution of the velocity gauge potential? The code seems not complete.

**Comment by HeFuxiang94** [comment]
> Does this PR only include a part of the solution of the velocity gauge potential? The code seems not complete.

Yes, this is only a part, I will gradually upload it. Here I just try to familiarize myself with the PR process.

File: source/module_basis/module_ao/ORB_table_phi.cpp [hunk 1]
```diff
@@ -170,6 +170,21 @@ void ORB_table_phi::cal_ST_Phi12_R
 			{
 				k1_dot_k2[ik] = n1.getPsi_k2(ik) * n2.getPsi_k2(ik);
 			}
+			
+			int tddft = 0;
```
Comment (dyzheng) [inline source/module_basis/module_ao/ORB_table_phi.cpp:174]: There is no setting for tddft parameter here, so the code block is useless now?

File: source/module_basis/module_ao/ORB_table_phi.cpp [hunk 2]
```diff
@@ -170,6 +170,21 @@ void ORB_table_phi::cal_ST_Phi12_R
 			{
 				k1_dot_k2[ik] = n1.getPsi_k2(ik) * n2.getPsi_k2(ik);
 			}
+
```
Comment (dyzheng) [inline source/module_basis/module_ao/ORB_table_phi.cpp:173]: please add unittest with gtest for function "ORB_table_phi::cal_ST_Phi12_R" before you modified it.

File: source/module_elecstate/potentials/H_TDDFT_pw.cpp [hunk 3]
```diff
@@ -198,6 +198,30 @@ double H_TDDFT_pw::cal_v_space_length_potential(double i)
 
 void H_TDDFT_pw::cal_v_space_velocity(std::vector<double> &vext_space, int direc)
 {
+	ModuleBase::TITLE("H_TDDFT_pw", "cal_v_space_velocity");
```
Comment (dyzheng) [inline source/module_elecstate/potentials/H_TDDFT_pw.cpp:201]: Please add annotations in H_TDDFT_pw.h with doxygen format and annotations should contain formula of cal_v_space_velocity.

Comment (Satinelamp) [reply source/module_elecstate/potentials/H_TDDFT_pw.cpp:201]: Does ABACUS support using Doxygen to generate the doc currently? I didn't find the configuration file for Doxygen.

**Comment by mohanchen** [comment]
No active answer, we will close the PR.

**Pull request closed without merging**